### PR TITLE
Fixing code comment for the put method of MultiMap

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

**Comment by AyberkSorgun** [comment]
@asimarslan could you review?

File: hazelcast/src/main/java/com/hazelcast/multimap/MultiMap.java [hunk 1]
```diff
@@ -111,7 +111,9 @@
      * @param key   the key to be stored
      * @param value the value to be stored
      * @return {@code true} if size of the multimap is increased,
-     * {@code false} if the multimap already contains the key-value pair
+     * {@code false} if the collection type is SET and multimap already
+     * contains the key-value pair, {@code false} otherwise if
+     * the collection type is LIST
```
Comment (ahmetmircik) [inline hazelcast/src/main/java/com/hazelcast/multimap/MultiMap.java:116]: For `LIST` type, method `put` returns true. To reword concisely we can have something like this:

```
{@code false} if the multimap already contained the key-value pair and doesn't allow duplicates
      
```

**Review by ahmetmircik — CHANGES_REQUESTED** [review]
please see my comment

**Comment by Serdaro** [comment]
Couldn't find the PR branch. Opened a new PR: https://github.com/hazelcast/hazelcast/pull/20723

**Pull request closed without merging**